src/test/java/com/cloudbees/jenkins/support/SupportActionTest.java

```diff
@@ -347,7 +345,7 @@ private ZipFile downloadSupportBundleByButtonClick() throws IOException, SAXExce
         AtomicReference<HtmlPage> pageRef = new AtomicReference<>(page);
 
         HtmlPage finalPage = page;
-        await().timeout(10, TimeUnit.HOURS).until(() -> {
+        await().timeout(10, TimeUnit.SECONDS).until(() -> {
```

Comment:
or simply
```suggestion
        await().until(() -> {
```
since this is the default timeout